src/Dotnet.Script.DependencyModel/ProjectSystem/ScriptProjectProvider.cs

```diff
@@ -74,6 +74,16 @@ private void LogProjectFileInfo(string pathToProjectFile)
 
         public string CreateProject(string targetDirectory, string defaultTargetFramework = "net46", bool enableNuGetScriptReferences = false)
```

Comment:
Should we mark this as obsolete too?

Reply:
this is actually used by us at runtime so it's needed